api/BaseAudioContext.json

```diff
@@ -665,12 +713,10 @@
               "version_added": "14"
             },
             "firefox": {
-              "version_added": "53",
-              "notes": "Originally implemented on <a href='https://developer.mozilla.org/docs/Web/API/AudioContext'><code>AudioContext</code></a> in Firefox 50."
+              "version_added": "50"
```

Comment:
This matches https://developer.mozilla.org/en-US/docs/Web/API/IIRFilterNode#browser_compatibility.